modules/tinymce/src/plugins/accordion/test/ts/browser/AccordionPluginTest.ts

```diff
@@ -167,6 +168,29 @@ describe('browser.tinymce.plugins.accordion.AccordionPluginTest', () => {
     TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
   });
 
+  it('TINY-12316: Toggle an accordion element not possible when disabled', () => {
```

Comment:
I think you can make a test for the default prevent as well. Just do a Mouse.click on the details element then you have a `editor.on('click', ..)` that captures the event object and you check is `isDefaultPrevented` is true on it.